Docs/Configuration.tex

```diff
@@ -5752,27 +5752,27 @@ \subsection{Output Properties}\label{uefioutputprops}
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Use builtin graphics output protocol renderer for console.
 
-  On some firmwares this may provide better performance or even fix rendering issues,
-  like on \texttt{MacPro5,1}. However, it is recommended not to use this option unless
+  On some firmware, such as on the \texttt{MacPro5,1}, this may provide better performance
+  or even fix rendering issues. However, this option is not recommended unless
   there is an obvious benefit as it may even result in slower scrolling.
 
 \item
   \texttt{IgnoreTextInGraphics}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Select firmwares output text onscreen in both graphics and
-  text mode. This is normally unexpected, because random text may appear over
+  \textbf{Description}: Some firmware output text onscreen in both graphics and
+  text mode. This is typically unexpected as random text may appear over
   graphical images and cause UI corruption. Setting this option to \texttt{true} will
-  discard all text output when console control is in mode different from \texttt{Text}.
+  discard all text output when console control is in a different mode from \texttt{Text}.
 
-  \emph{Note}: This option only applies to \texttt{System} renderer.
+  \emph{Note}: This option only applies to the \texttt{System} renderer.
 
 \item
   \texttt{ReplaceTabWithSpace}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Some firmwares do not print tab characters or even everything
-  that follows them, causing difficulties or inability to use the UEFI Shell builtin
+  \textbf{Description}: Some types of firmware do not print tab characters or everything
+  that follows them, causing difficulties in using the UEFI Shell's builtin
```

Comment:
It is not a living being, is `'s` actually allowed?

Reply:
There is no consistent rule, some people tend to avoid it, but it works.